### DISCUSSION TITLE
Redirect conversation to human

### DISCUSSION THREAD

**Discussion by FacuBozzi** [FlowiseAI/Flowise#3628] [Q&A]
I have a situation where customers would talk to an AI chatbot, but if the user asks to speak to a human, a real person (from support) would take over the discussion and answer the customer's questions in the chat. Is this possible in Flowise?

Thanks in advance!

**Comment by bernejorge** [comment]
If you call Flowise through API, you can avoid calling the API when a client wants to talk to a human.
You can then insert the human's response into the "chat_message" table. Adding the value 'apiMessage' to the column role. 
I haven't tried it, it's just an idea.
